CHANGELOG.md

```diff
@@ -4,6 +4,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.25.0] - 2022-11-16
+
+This release resolves the `parity-util-mem crate` several version guard by updating substrate related dependencies which makes
+it possible to have other substrate dependencies in tree again along with subxt.
+
+In addition the release has several API improvements in the dynamic transaction API along with that subxt now compiles down to WASM.
```

Comment:
Love the casual mention of the most important feature of all: Wasm compilation :heart: